A6lib.cpp

```diff
@@ -1,319 +1,368 @@
-#include <Arduino.h>
-#include <SoftwareSerial.h>
-#include "A6lib.h"
+#include <stdarg.h>
 
+#include "A6lib.h"
 
-/////////////////////////////////////////////
-// Public methods.
-//
+#define countof(a) (sizeof(a) / sizeof(a[0]))
+#define A6_CMD_TIMEOUT 2000
 
-A6lib::A6lib(int transmitPin, int receivePin) {
-#ifdef ESP8266
-    A6conn = new SoftwareSerial(receivePin, transmitPin, false, 1024);
-#else
-    A6conn = new SoftwareSerial(receivePin, transmitPin, false);
+#ifdef DEBUG
+#ifndef DEBUG_PORT
+#	define DEBUG_PORT Serial
```

Comment:
Should there be a tab between `#` and `define`?

Reply:
Just a habit :)
Will remove it